cache/caffeine/src/test/java/io/quarkus/ts/cache/caffeine/cache/caffeine/CaffeineCacheIT.java

```diff
@@ -72,6 +73,32 @@ public void shouldGetTheSameValueForSamePrefixesWhenGettingValueFromPath(String
                 "Value was equal which means @CacheKey didn't work");
     }
 
+    /**
+     * Ensure that failed unis are not cached
+     */
+    @Tag("https://github.com/quarkusio/quarkus/pull/39762")
+    @ParameterizedTest
+    @ValueSource(strings = { RESOURCE_REACTIVE_API_PATH + "/failing-value" })
```

Comment:
Afaik are we need this? `@ValueSource` is used for more parameters but I see only one

Reply:
Yes, it's not needed here. Removing it, thanks.